README.md

```diff
@@ -37,6 +37,33 @@ doctrine:
                     engine: MyISAM
 ```
 
+## MySQL Strict Mode
+
+It is recommended to run MySQL in "strict mode" to prevent corrupt or truncated
+data and to guarantee data integrity. As of Contao 4.12, the install tool shows
+a warning if the database server is not running in strict mode. To enable it,
+add the following to your `my.cnf` file or make sure that the setting is adjusted
+accordingly:
+
+```
+[mysqld]
+…
+sql_mode="STRICT_TRANS_TABLES"
+…
+```
+
+If the setting cannot be enabled on your server, please configure the connection
+options in your `config/config.yml` file:
+
+```yml
+doctrine:
+    dbal:
+        connections:
+            default:
+                options:
+                    1002: "SET SESSION sql_mode=(SELECT CONCAT(@@sql_mode, ',TRADITIONAL'))"
```

Comment:
This option is `1002` only if PDO is used. For mysqli the option is named `3`, see https://github.com/contao/contao/blob/f1b3c56856666c38fe080fd04e00797b8fbed310/installation-bundle/src/InstallTool.php#L297

Reply:
Would you specify that here as well?

The warning in the install tool also only shows this option:

![mysql-strict-mode](https://user-images.githubusercontent.com/1218766/130322519-2f147e1d-9131-41fe-9a28-ed167a49685c.png)

Reply:
@xchs the key is automatically determined by the install tool. So depending on your connection, it will either show `1002` or `3`.

Reply:
Thank you, all right. Then it will be a bit more cumbersome to document this accordingly... :)